### PR TITLE
OCM-8416 | ci: Automate 64095

### PR DESCRIPTION
**Pull request by jameszwang**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (OCM-xxxx)*:
Fixes #[OCM-8416](https://issues.redhat.com//browse/OCM-8416)

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [ x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [ x] Subject and description added to both, commit and PR.
- [ x] Relevant issues have been referenced.

### CODE REVIEW

**Comment by radtriste** [comment]
CI failures are known and under investigation

File: tests/e2e/negative_day_one_test.go [hunk 1]
```diff
@@ -841,4 +841,24 @@ var _ = Describe("Negative Tests", Ordered, func() {
 			restoreSubnetTagging()
 		})
 	})
+
+	var _ = Describe("The EOL OCP version validation", ci.NonHCPCluster, ci.Day1Negative, func() {
```
Comment (radtriste) [inline tests/e2e/negative_day_one_test.go:845]: Can you test it against HCP as well ? It should work IMO

Comment (jameszwang) [reply tests/e2e/negative_day_one_test.go]: @radtriste the test can work with HCP cluster, remove d ci.NonHCPCluster label.

**Review by radtriste — APPROVED** [review]
/lgtm

**Pull request merged**